api/services/AssetService.ts

```diff
@@ -0,0 +1,68 @@
+import { ServiceOptions, Service } from 'ts-framework-common';
+import BitcapitalService from './BitcapitalService';
+import { User } from '../models';
+export interface AssetServiceOptions extends ServiceOptions{
+
+}
+
+export default class AssetService extends Service{
+    protected static instance: AssetService;
+    private static assetId: string = 'BOBD';
+
+    constructor(options: AssetServiceOptions) {
+        super(options);
+    }
+
+    public async creditAssetForUser(receiverId:string, amount:number){
+        const bitcapitalClient = BitcapitalService.getInstance({}).bitcapital;
+        const receiver: User = await User.findById(receiverId);
+        
+        return await bitcapitalClient.assets().emit({
+            amount: amount.toString(),
+            destination: receiver.bitcapitalId,
+            id: AssetService.assetId
+        });
+    }
+
+    public async debitAssetFromUser(senderId:string, amount:number){
+        const bitcapitalClient = BitcapitalService.getInstance({}).bitcapital;
+        const sender: User = await User.findById(senderId);
+
+        return await bitcapitalClient.assets().destroy({
+            amount: amount.toString(),
+            source: sender.bitcapitalId,
+            id: AssetService.assetId
+        });
+    }
+    
+    public static getInstance(options: AssetServiceOptions) {
```

Comment:
```suggestion
    public static getInstance(options: AssetServiceOptions = {}) {
```

Você pode usar `getInstance(options: AssetServiceOptions = {})` aqui e nos outros services para que options seja por padrão `{}` e você possa fazer `AssetService.getInstance()` no lugar do atual `AssetService.getInstance({})`

Reply:
Boooa

Reply:
Fixed